src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -114,7 +114,7 @@ class Docker implements Serializable {
         
         public <V> V inside(String args = '', Closure<V> body) {
             docker.node {
-                if (docker.script.sh(script: "docker inspect -f . ${id}", returnStatus: true) != 0) {
+                if (docker.script.sh(script: "docker inspect --type image ${id}", returnStatus: true) != 0) {
```

Comment:
Of course you are totally right. it was just the lack of the type.
Just tested and validated your suggestion. It works like a charm in UCP 1.11.1-cs2
And in UCP 2.0.1 it even works without the --type arg